content/api/inline-seeds.apib

```diff
@@ -0,0 +1,511 @@
+
+    + Headers
+
+        Authorization: 14ac5499cfdd2bb2859e4476d2e5b1d2bad079bf
+
+    + Body
+
+        ```json
+        {
+          "threshold": 1000,
+          "duration": 20,
+          "reset": 24
+        }
+        ```
++ Response 422 (application/json)
+
+    + Body
+
+        ```json
+        {
+            "errors": [
+                {
+                    "message": "Either match or exclude has to be specified"
+                }
+            ]
+        }
+        ```
+
+### Update Seed Config [PUT /inline-seeds/config]
+
+Make updates to config details for the authenticated account.
+Only one seed config with 5 match rules max is currently supported.
+Auth context (from API key) provides customer and subaccount ids.
+
++ Request - Remove redundant rule (application/json)
+
+    + Headers
+
+        Authorization: 14ac5499cfdd2bb2859e4476d2e5b1d2bad079bf
+
+    + Body
+
+        ```json
+        {
+          "threshold": 1000,
+          "duration": 20,
+          "reset": 24,
+          "match": [
+            "my-*"
+          ]
+        }
+        ```
+
++ Response 200
+    + Body
+
+        ```json
+        {
+          "results": {
+            "threshold": 1000,
+            "duration": 20,
+            "reset": 24,
+            "match": [
+              "my-*"
+            ],
+            "exclude": []
+          }
+        }
+        ```
+
++ Request - Too many matches (application/json)
+
+    + Headers
+
+        Authorization: 14ac5499cfdd2bb2859e4476d2e5b1d2bad079bf
+
+    + Body
+
+        ```json
+        {
+          "threshold": 1000,
+          "duration": 20,
+          "reset": 24,
+          "match": [
+            "fred-*",
+            "wilma-*",
+            "pebbles-*",
+            "barney-*",
+            "betty-*",
+            "bam-bam-*"
+          ]
+        }
+        ```
+
++ Response 422 (application/json)
+
+    + Body
+
+        ```json
+        {
+            "errors": [
+                {
+                    "message": "Match rule limit exceeded (found 6, max is 5)"
+                }
+            ]
+        }
+        ```
+
++ Request - Both match and exclude (application/json)
+
+    + Headers
+
+        Authorization: 14ac5499cfdd2bb2859e4476d2e5b1d2bad079bf
+
+    + Body
+
+        ```json
+        {
+          "threshold": 1000,
+          "duration": 20,
+          "reset": 24,
+          "match": [
+            "fred-*",
+            "wilma-*"
+          ],
+          "exclude": [
+            "barney-*",
+            "betty-*"
+          ]
+        }
+        ```
+
++ Response 422 (application/json)
+
+    + Body
+
+        ```json
+        {
+            "errors": [
+                {
+                    "message": "Incompatible rule type combination: 'match' and 'exclude' are mutually exclusive."
+                }
+            ]
+        }
+        ```
+
+### Check Campaign Eligibility [GET /inline-seeds/match?id={campaign_id}]
```

Comment:
is this customer facing endpoint?

Reply:
I'm not sure that it strictly is. There was some discussion about the UI using it at one point. If it isn't used I'd like to leave it undocumented so that we can make alterations freely.

I can see it being useful for customers testing their config, though

Reply:
I've removed the match endpoint. Thank you for calling that out